code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -565,7 +565,6 @@
 		new /obj/item/grenade/spawnergrenade/buzzkill(src)
 
 /obj/item/storage/box/syndie_kit/sleepytime/PopulateContents()
-	new /obj/item/clothing/under/syndicate/bloodred/sleepytime(src)
```

Comment:
If you're just for whatever reason removing the pjs, just remove the kit, okay?

Reply:
I removed the pjs 'cause they don't exist lol, but yeah, sure.

Reply:
The PJ's are very important after a hard day of nuking...